core/src/main/scala/org/bitcoins/core/protocol/ln/currency/MilliSatoshis.scala

```diff
@@ -62,27 +63,63 @@ sealed abstract class MilliSatoshis extends NetworkElement {
   }
 
   def ==(ms: MilliSatoshis): Boolean = {
-    toLnCurrencyUnit == ms.toLnCurrencyUnit
+    toBigInt == ms.toBigInt
   }
 
   def !=(ms: MilliSatoshis): Boolean = {
-    toLnCurrencyUnit != ms.toLnCurrencyUnit
+    toBigInt != ms.toBigInt
   }
 
   def >=(ms: MilliSatoshis): Boolean = {
-    toLnCurrencyUnit >= ms.toLnCurrencyUnit
+    toBigInt >= ms.toBigInt
   }
 
   def >(ms: MilliSatoshis): Boolean = {
-    toLnCurrencyUnit > ms.toLnCurrencyUnit
+    toBigInt > ms.toBigInt
   }
 
   def <(ms: MilliSatoshis): Boolean = {
-    toLnCurrencyUnit < ms.toLnCurrencyUnit
+    toBigInt < ms.toBigInt
   }
 
   def <=(ms: MilliSatoshis): Boolean = {
-    toLnCurrencyUnit <= ms.toLnCurrencyUnit
+    toBigInt <= ms.toBigInt
+  }
+
+  def +(ms: MilliSatoshis): MilliSatoshis = {
+    MilliSatoshis(toBigInt + ms.toBigInt)
+  }
+
+  def -(ms: MilliSatoshis): Try[MilliSatoshis] = {
```

Comment:
we aren't being consistent here with the return types, should be `MilliSatoshis` rather than `Try[MilliSatoshis]`